Ports/jdupes/package.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='jdupes'
+version='1.26.1'
+files="https://github.com/jbruchon/jdupes/archive/refs/tags/v${version}.tar.gz jdupes-${version}.tar.gz 09153824320c65ad529f8f97cd3b7e792c50e9f9018192ea1a76f2e33a196225"
+auth_type='sha256'
+workdir="jdupes-${version}"
+depends=('libjodycode')
+makeopts+=('DISABLE_DEDUPE=1')
```

Comment:
this port has the same issue as the previous one, re `UNAME_S`

Reply:
Addressed. Does this need any further changes?